packages/experiments/src/components/Toggle/__snapshots__/Toggle.view.test.tsx.snap

```diff
@@ -12,6 +12,22 @@ exports[`ToggleView renders toggle correctly 1`] = `
         font-size: 14px;
         font-weight: 400;
         margin-bottom: 8px;
+        outline: transparent;
+        position: relative;
+      }
+      &::-moz-focus-inner {
+        border: 0;
+      }
+      .ms-Fabric--isFocusVisible &:focus:after {
+        border: 1px solid #ffffff;
+        bottom: 1px;
+        content: "";
+        left: 1px;
+        outline: 1px solid #666666;
+        position: absolute;
+        right: 1px;
+        top: 1px;
+        z-index: 1;
```

Comment:
Hmm what is the cause for these changes? (I would expect this file to be mostly the same...)

Reply:
I realized what I was doing here. It's fixed, the snapshot should be the same as it was before.